src/sentry/utils/email/message_builder.py

```diff
@@ -250,7 +269,7 @@ def send_async(
             send_email_task = send_email.delay
             if SiloMode.get_current_mode() == SiloMode.CONTROL:
                 send_email_task = send_email_control.delay
-            safe_execute(send_email_task, message=message)
+            safe_execute(send_email_task, message=message_to_dict(message))
```

Comment:
This looks like it would work. I wonder if we shouldn't also move the logic from `get_built_messages` into the `send_email_task` functions as well 🤔 It looks like `MessageBuilder` would be simpler to JSON encode than django's mail objects are.

Reply:
We could do that, but it would require changing the email API quite a bit I think. Mostly that's because `get_built_messages` returns a list of messages, each of which are executed in a separate task. So we would need to change the API of the tasks, or add a new task that takes a message builder and migrate the code onto that new task.

Either way I think that is maybe outside the scope of this change.

Reply:
> Mostly that's because get_built_messages returns a list of messages, each of which are executed in a separate task. 

I missed that part of the logic. That behavior would require larger changes to be made.